filters/ThirdParty/filter_238_PolskiFiltrAdg/template.txt

```diff
@@ -1,2 +1 @@
 !
-@include "https://raw.githubusercontent.com/MajkiIT/polish-ads-filter/master/polish-adblock-filters/adblock_adguard.txt" /exclude="../../exclusions.txt"
```

Comment:
We need to change the file contents here (mention that it is now included into the main subscription)

Reply:
but `filter.txt` contains `! included to the main Official Polish filters for AdBlock, uBlock Origin & AdGuard`

Reply:
It might be replaced when the filter is being rebuilt. Not that it will happen, but anyway just in case we'd better have it

Reply:
Got it. Moved that comment to `template.txt`